klimenko/Lesson3_4_Klimenko/src/main/java/entity/RealEstate.java

```diff
@@ -0,0 +1,42 @@
+package entity;
+
+public abstract class RealEstate {
```

Comment:
Why abstract? 
And better rename to Estate, so you can use type instead of typeEstate, and id instead of estateId